doc/sentence_length_diffs_inference_report.Rmd

```diff
@@ -25,85 +25,86 @@ ci_95 <- readRDS(paste(params$results_dir_path,"/ci_95.rds", sep=""))
+All other columns were dropped. While there were mo missing values, some sentences are of indeterminate length, and could not be used in the analysis. We also dropped all the rows with indeterminate sentences. At the end of data processing, the count of observations in either group was `r counts$Count[2]` and `r format(counts$Count[1])`. This provided a large enough sample size.
 
 ```{r observations_group}
 knitr::kable(counts, caption = "Table 1. Number of observations in each group.", col.names = c("Group",
                            "Count of Offenders"))
 
                 
 ```
-<br />
-To decide what kind of measure of central tendency to apply in our hypothesis test, we looked at the distribution of the sentence lengths of either group (orange - non-indigenous, blue - indigenous) using a density plot. The distributions for both groups were right skewed, so we decided to use the median as our parameter as it is less affected by the outliers. 
+
+<br /> To decide what kind of measure of central tendency to apply in our hypothesis test, we looked at the distribution of the sentence lengths of either group (orange - non-indigenous, blue - indigenous) using a density plot. The distributions for both groups were right skewed, so we decided to use the median as our parameter because it is less affected by the outliers in the data.
 
 ```{r desnity, fig.width=7, fig.height=3, fig.cap = "Figure 1. The distribution of the aggregate sentence length of offenders by racial group.", out.width='100%'}
 knitr::include_graphics('../results/eda-densities.png')
 ```
 
-Looking at the boxplot, we noted the large overlap in the confidence intervals between the two groups, which at a glance indicated that a statistically significant difference was unlikely. 
+Looking at the box-plots, we noted the large overlap in the confidence intervals between the two groups, which at a glance indicated that a statistically significant difference was unlikely.
 
 ```{r boxplots, fig.width=10, fig.height=3, fig.cap = "Figure 2. The boxplot of the sentence length of offenders by racial group. The marker at the center of the box corresponds to the median.", out.width='100%'}
 
 knitr::include_graphics('../results/eda-box_plots.png')
 ```
 
-We had a large enough sample size to apply a permutation test as the toal sample size was `r format(counts$Count[1] + counts$Count[2])`. We carried out the permutation test with r N_REPEATS under the null hypothesis, computing and comparing the median for each of the two groups.
+We had a large enough sample size to apply a permutation test as the total sample size was `r format(counts$Count[1] + counts$Count[2])`. We carried out the permutation test with `r N_REPEATS` under the null hypothesis, computing and comparing the median for each of the two groups.
```

Comment:
I do not think the `r N_REPEATS` is parameterized so I think it is better to replace it with the raw value _5000_.